scenario-definitions.yaml

```diff
@@ -50,3 +50,12 @@ job_templates:
       INSTALL: "1"
       INSTALL_ONLY: "1"
       UPDATE: "0"
+  openqa_install_nginx:
+    product: openqa-*-dev-x86_64
+    machine: 64bit-2G
+    settings:
+      OPENQA_HOST: "http://openqa.opensuse.org"
```

Comment:
why do we need this setting?

Reply:
idk - I copied the section above which has the same setting.
@baierjan I guess you added that section. Any special reason for this line?

Reply:
If I recall correctly, the default https protocol failed in some step (maybe due to the weird proxy setup on o3?), this was a workaround recommended by @okurz (https://suse.slack.com/archives/C02AJ1E568M/p1682587547710399?thread_ts=1682587184.320269&cid=C02AJ1E568M)

Reply:
Well, that proxy setup is over. Let's try without this setting :) @asdil12 can you please remove it in another commit from all mentions in this file